docs/.vitepress/theme/custom.css

```
@@ -1,73 +1,100 @@
+
+  /* ── Text — dark mode ── */
+  --vp-c-text-1: #f0f0f2;
+  --vp-c-text-2: #a0a4a8;
+  --vp-c-text-3: #6c7073;
 }
 
+/* ── Typography — nexu font stack ── */
+
 body {
-  font-family: "Avenir Next", "Segoe UI", sans-serif;
+  font-family: 'Manrope', 'Inter', 'PingFang SC', -apple-system, 'Noto Sans SC', sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)